tests/test_new_client.py

```diff
@@ -299,6 +297,56 @@ def test_tools_from_crewai(client):
     assert expected_content in func(website_url=simple_webpage_url)
 
 
+def test_tools_from_langchain(client):
+    # create langchain tool
+    from langchain_community.tools import WikipediaQueryRun
+    from langchain_community.utilities import WikipediaAPIWrapper
+
+    from letta.schemas.tool import Tool
+
+    api_wrapper = WikipediaAPIWrapper(top_k_results=1, doc_content_chars_max=100)
+    langchain_tool = WikipediaQueryRun(api_wrapper=api_wrapper)
+
+    # Translate to memGPT Tool
+    tool = Tool.from_langchain(langchain_tool, additional_module_attr_import_map={"langchain_community.utilities": "WikipediaAPIWrapper"})
```

Comment:
maybe `required_module_location`?

Reply:
I might keep this, it's pretty long, will swap the import to the beginning though.